src/CoreBundle/DataAccess/DataAccessCmsTblConfInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\DataAccess;
+
+use ChameleonSystem\CoreBundle\DataModel\TableConfigurationDataModel;
+use ChameleonSystem\SecurityBundle\Voter\CmsPermissionAttributeConstants;
+
+interface DataAccessCmsTblConfInterface
+{
+    public const PERMISSION_MAPPING = [
+        CmsPermissionAttributeConstants::TABLE_EDITOR_NEW => 'cms_role_mlt',
+        CmsPermissionAttributeConstants::TABLE_EDITOR_EDIT => 'cms_role1_mlt',
+        CmsPermissionAttributeConstants::TABLE_EDITOR_DELETE => 'cms_role2_mlt',
+        CmsPermissionAttributeConstants::TABLE_EDITOR_EDIT_ALL => 'cms_role3_mlt',
+        CmsPermissionAttributeConstants::TABLE_EDITOR_ACCESS_ALL => 'cms_role6_mlt',
+        CmsPermissionAttributeConstants::TABLE_EDITOR_NEW_LANGUAGE => 'cms_role4_mlt',
+        //CmsPermissionAttributeConstants::WORKFLOW_PUBLISH => 'cms_role5_mlt',
+        //CmsPermissionAttributeConstants::VERSIONING => 'cms_role7_mlt',
```

Comment:
The tables still exists - but the permission is not used anywhere (and per default disabled). But since they are still there, I now added them as constants